docusaurus.config.js

```diff
@@ -10,71 +10,93 @@ module.exports = {
   organizationName: 'jellyfin',
   projectName: 'jellyfin.org',
   themeConfig: {
+    colorMode: {
+      defaultMode: 'light',
+      disableSwitch: true,
+      respectPrefersColorScheme: false
+    },
     navbar: {
-      title: 'My Site',
       logo: {
-        alt: 'My Site Logo',
+        alt: 'Jellyfin logo',
         src: 'img/logo.svg',
       },
       items: [
+        { to: "blog", label: "Blog", position: "right" },
```

Comment:
A lot of your additions use double quotes while the document used only single quotes before.